### DISCUSSION TITLE
Static files and caching

### DISCUSSION THREAD

**Discussion by snelgrove** [beginner-corp/begin-community#91] [Q&A] [answered]
After enabling `fingerprint external`, I have a static site running with cached images and non-cached html files. It's cool that I can use @proxy to serve this without even writing a lambda!

What if this were a blog and I wanted to publish an RSS feed? `feed.xml` shouldn't have a fingerprinted filename or a super-long cache, but it seems that only .html files are served fresh. Is there any built-in way to omit a file from fingerprint/cache or would this require it's own route and lambda that overrides the @proxy?

**Comment by brianleroux** [comment, marked as answer]
hey @snelgrove; check out the `ignore` flag https://arc.codes/docs/en/reference/app.arc/static !

**Reply by snelgrove** [reply]
Thanks for the reply! I have now tried the ignore flag, but from what I can see, `ignore` will completely ignore a file and won't deploy it at all. What I'm trying to do is deploy files such as `feed.xml` or `sitemap.xml` without a fingerprinted filename and without cache. 

It seems that html files are always excluded from caching, but it would be cool if I could whitelist other files to bypass cache headers.

**Reply by ryanblock** [reply]
If you'd like full control over your headers, I'd suggest the happy path would simply be delivering those files from an http endpoint!

**Reply by snelgrove** [reply]
Hey @ryanblock, a new endpoint is on my backlog. I was just wondering if there was some config I has missed, since you guys have covered so many requirements. Anyway, arc is fun so I don't mind building a new route 🤓
